pkg/syncer/admissionhandler/validatepvcannotationforvolumehealth.go

```diff
@@ -26,6 +26,10 @@ func validatePVCAnnotationForVolumeHealth(ctx context.Context, request admission
 	username := request.UserInfo.Username
 	isCSIServiceAccount := validateCSIServiceAccount(request.UserInfo.Username)
 	log.Debugf("validatePVCAnnotationForVolumeHealth called with the request %v by user: %v", request, username)
+	if request.Operation == admissionv1.Delete {
+		// PVC volume health annotation validation is not required for delete PVC calls
+		return admission.Allowed("")
+	}
 	newPVC := corev1.PersistentVolumeClaim{}
 	if err := json.Unmarshal(request.Object.Raw, &newPVC); err != nil {
 		reason := "skipped validation when failed to deserialize PVC from new request object"
```

Comment:
Can we log err as well in all such cases where unmarshal is failing? It will help in debugging issues.

Reply:
done.